src/rf/js/src/home/components/processing-block.js

```diff
@@ -17,7 +17,7 @@ var LayerStatusComponent = React.createBackboneClass({
             chunkClass = this.pendingClass,
             mosaicClass = this.pendingClass,
             completeClass = this.pendingClass,
-            actionLinks = (<a className="text-danger">Cancel</a>),
+            actionLinks = (<a href="javascript:;" onClick={this.deleteLayer} className="text-danger">Cancel</a>),
```

Comment:
They're just links without destinations. How is that more wrong than having hash links and then intercepting the navigation and suppressing it? This is just making a button out of something that is not a button. Maybe we should just make a button out of it.